packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -318,8 +320,27 @@ export class AppManagementClient implements DeveloperPlatformClient {
     }
   }
 
-  async devStoresForOrg(_orgId: string): Promise<OrganizationStore[]> {
-    return []
+  async devStoresForOrg(orgId: string): Promise<OrganizationStore[]> {
+    const base64Id = encodedGidFromId(orgId)
+    const variables: DevStoresQueryVariables = {organizationId: base64Id}
```

Comment:
This is a pattern we've established intentionally, to make sure the variables stay consistent with what we have in the graphql file. Otherwise, nothing in our system validates that the types of the variables are correct.

Note this convention was established in response to an actual production incident, where we changed the name of a variable and nothing caught it.

Reply:
I see what you mean now -- I think this is a point in favour of (re-)picking up codegen and something based on https://github.com/dotansimha/graphql-typed-document-node that ties the input/output types together.